### ISSUE TITLE
libfreeaptx-0.1.1: advertises wrong version numbers

### ISSUE THREAD

**Issue by tgurr** [iamthehorker/libfreeaptx#1]
```
$ freeaptxdec --help
aptX decoder utility 0.2.0 (using libfreeaptx 0.2.0)
```
```
$ freeaptxenc --help
aptX encoder utility 0.2.0 (using libfreeaptx 0.2.0)
```

freeaptx.h
```
#define freeaptx_MAJOR 0
#define freeaptx_MINOR 2
#define freeaptx_PATCH 0
```

**Comment by regularhunter** [comment]
Should be fixed in commit c176b7d. Let me know if that doesn't work for you.

**Comment by tgurr** [comment]
> Should be fixed in commit [c176b7d](https://github.com/iamthehorker/libfreeaptx/commit/c176b7de9c2017d0fc1877659cea3bb6c330aafa). Let me know if that doesn't work for you.

Thanks, LGTM: `aptX encoder utility 0.1.1 (using libfreeaptx 0.1.1)`
